ec2/templates/user_data_jenkins_docker.sh.tpl

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+echo ECS_CLUSTER=${jenkins_cluster_name} > /etc/ecs/ecs.config
+chown 1000:1000 /var/run/docker*
```

Comment:
This will trigger a restart of Jenkins if I add a line break in so I've left it.

Reply:
If you change anything in the file, it changes the hash and causes a restart. I should have snuck it in when I was breaking Jenkins before but I forgot. If you can live with it for a bit, I'll try and add one next time we get a new AMI